### DISCUSSION TITLE
Img2img

### DISCUSSION THREAD

**Discussion by CallMeWhatever** [dobrado76/Stable-Diffusion-Unity-Integration#11] [Q&A]
Might be a stupid question but can this be used in a similar way to a blender stable diffusion integration? For example render an image of a cube in unity and then use it with a promt for img2img generating inside Unity

**Comment by dobrado76** [comment]
Indeed, the Img2Img does not care where image is from. You can have a cube (or any 3d model) and camera that renders it to a texture and feed that to the Img2Img. I have not experimented with that but if the current setup is not sufficient, one could easily modify the StableDiffusionImage2Image component to accept a RenderTexture instead of a Texture2D as input (or create a new StableDiffusionRenderTexture2Image which would be a minor variation of the StableDiffusionImage2Image component).

**Reply by CallMeWhatever** [reply]
Awesome, and Thanks! Im quite new to this stuff and come mostly from 3d softwares and bit of UE but I'll give it a try :)
